crap-advisor-frontend/src/core/components/Map/index.jsx

```diff
@@ -1,29 +1,39 @@
-import React from 'react';
-import {YMaps, Map as YMap, ZoomControl, Placemark} from '@pbe/react-yandex-maps';
+import React, {useEffect, useState} from "react";
 import "./index.css"
-import restroomImage from '../../../assets/restroom.png'
+import {MapContainer, TileLayer, ZoomControl} from "react-leaflet";
+import RestroomMarker from "../RestroomMarker";
+
+const apiUrl = "http://localhost:8082"
+const defaultPosition = [53.21176, 50.18394]
+const defaultZoom = 13
```

Comment:
Keep all of your variables inside a component

Reply:
done